jobs/involuntary-dissolutions/requirements.txt

```diff
@@ -26,5 +26,5 @@ six==1.15.0
 urllib3==1.26.11
 requests==2.25.1
 cachelib==0.1.1
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+legal_api[nats] @ git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
```

Comment:
@leodube-aot This should work now if you run into problem on installing all legal-api dependencies. 👀